server/planning/content_api/events/__init__.py

```diff
@@ -8,7 +8,46 @@
 # AUTHORS and LICENSE files distributed with this source code, or
 # at https://www.sourcefabric.org/superdesk/license
 
+from typing import Optional
+from superdesk.flask import g
+from pydantic import BaseModel
+from eve.utils import ParsedRequest
 from superdesk.core.module import Module
-from .event import content_api_event_resource_config
+from superdesk.core.web import EndpointGroup
+from werkzeug.datastructures import MultiDict
+from superdesk.core.types import Request, Response
+from .event import content_api_event_resource_config, ContentAPIEventService
+from planning.content_api.utils import ALLOWED_PARAMS, APIListParams
```

Comment:
APIListParams is not that descriptive. How about `PlanningCAPIParams` or something like that